src/pages/About/AboutResume.jsx

```diff
@@ -1,59 +1,78 @@
 /* eslint-disable react/no-unknown-property */
 import { motion } from 'framer-motion';
+import { Link } from 'react-router-dom';
+import { BsFillArrowRightCircleFill } from 'react-icons/bs';
+
 import Socials from '../../components/Socials';
-import { images } from '../../constants';
 
-const AboutResume = () => {
-  console.log('test');
-  return (
-    <div className="animate-slideright flex flex-col md:flex-row mb-8 px-8 pb-10 md:pb-0" id="resume">
-      <div className="w-full">
-        <div className="flex items-center gap-4 mb-12">
-          <h2 className="whitespace-nowrap uppercase tracking-[.4em] font-semibold">About Me</h2>
-          <motion.hr
-            initial={{ width: 0 }}
-            animate={{ width: '100%' }}
-            transition={{ duration: 1, ease: 'linear' }}
-            className="border border-gray-200"
-          />
+const AboutResume = () => (
+  <motion.div
+    whileInView={{ opacity: [0, 1], x: [-50, 0] }}
+    transition={{ duration: 1, staggerChildren: 0.5 }}
+    className="animate-slideright flex flex-col md:flex-row mb-8 px-8 pb-8 md:pb-0"
+    id="about"
+  >
+    <div className="w-full">
+      <div className="flex items-center gap-4 mb-12">
+        <h2 className="whitespace-nowrap uppercase tracking-[.4em] font-semibold">About Me</h2>
+        <motion.hr
+          initial={{ width: 0 }}
+          animate={{ width: '100%' }}
+          transition={{ duration: 1, ease: 'linear' }}
+          className="border border-gray-200"
+        />
+      </div>
+      <div className="flex text-sm flex-col gap-4">
+        <div className="w-full lg:hidden flex-col items-center animate-slowfade gap-4 uppercase">
+          <div className="relative flex items-center justify-center">
+            <h2 className="text-4xl sm:text-5xl tracking-[.5em] whitespace-nowrap text-gray-300 dark:text-gray-600 opacity-60">Resume</h2>
+            <h2 className="absolute text-xl sm:text-2xl font-semibold tracking-[.4em] whitespace-nowrap text-gray-800 dark:text-gray-200">About <span className="text-blue-500">Me</span></h2>
+          </div>
```

Comment:
These styles and some others seem repetitive in some places. Create a reusable style object in a separate file and use it wherever possible. 

Avoid writing the same long class names that make your code look less clean. You will also find anything quickly for any future updates with the reusable styles approach.